static/partition-view.html

```diff
@@ -40,6 +40,10 @@
       <div class="logo">
         <img src="/icons/logo_ocsf.svg">
         <span>Learn OCaml</span>
+        <select id="Hide tokens">
```

Comment:
In practice, you shouldn't use a capital and a space in an HTML5 `id`. Maybe rather:

```suggestion
        <select id="learnocaml-select-student-info">
```